nx/lib/nx/binary_backend.ex

```diff
@@ -841,6 +841,24 @@ defmodule Nx.BinaryBackend do
   @impl true
   def bitwise_not(out, tensor), do: element_wise_unary_op(out, tensor, &:erlang.bnot/1)
 
+  @impl true
+  def is_nan(out, %{type: {t, n_bits}}) when t in [:u, :s] do
+    # integers cannot represent nans, so we can just create
+    # a zero boolean tensor
+
+    size = Tuple.product(out.shape) * div(n_bits, 8)
```

Comment:
```suggestion
    size = Nx.size(out.shape) * div(n_bits, 8)
```

That seems to be what we use in this module.